File: EmpleadoTest.cs
using System;

public class EmpleadoTest
{
    public static void Main(string[] args)
    {
        Empleado empleado1 = new Empleado();
        Empleado empleado2 = new Empleado();

        Console.WriteLine("Ingresa el nombre del empleado 1: ");
        empleado1.nombre = Console.ReadLine();

        Console.WriteLine("Ingresa el apellido paterno: ");
        empleado1.apellido = Console.ReadLine();

        Console.WriteLine("Ingresa el salario mensual: ");
        empleado1.salario = double.Parse(Console.ReadLine());

        if (empleado1.salario < 0)
            empleado1.salario = 0.0;


        Console.WriteLine("\nIngresa el nombre del empleado 2: ");
        empleado2.nombre = Console.ReadLine();

        Console.WriteLine("Ingresa el apellido paterno: ");
        empleado2.apellido = Console.ReadLine();

        Console.WriteLine("Ingresa el salario mensual: ");
        empleado2.salario = double.Parse(Console.ReadLine());

        if (empleado2.salario < 0)
            empleado2.salario = 0.0;


        Console.WriteLine("\nSalario anual de "
            + empleado1.nombre + ": "
            + empleado1.obtenerSalarioAnual());

        Console.WriteLine("Salario anual de "
            + empleado2.nombre + ": "
            + empleado2.obtenerSalarioAnual());


        empleado1.aumentarSalario();
        empleado2.aumentarSalario();


        Console.WriteLine("\nDespués del aumento del 10%:");

        Console.WriteLine("Salario anual de "
            + empleado1.nombre + ": "
            + empleado1.obtenerSalarioAnual());

        Console.WriteLine("Salario anual de "
            + empleado2.nombre + ": "
            + empleado2.obtenerSalarioAnual());
    }
}

File: EmpleadoTest.cpp
#include <iostream>
#include <string>

using namespace std;

class Empleado {

    public:

    string nombre;
    string apellido;
    double salario;

    Empleado() {

        nombre = "";
        apellido = "";
        salario = 0.0;
    }

    double obtenerSalarioAnual() {

        return salario * 12;
    }

    void aumentarSalario() {

        salario = salario * 1.10;
    }

};

int main() {

    Empleado empleado1;
    Empleado empleado2;

    cout << "Ingresa el nombre del empleado 1: ";
    cin >> empleado1.nombre;

    cout << "Ingresa el apellido paterno: ";
    cin >> empleado1.apellido;

    cout << "Ingresa el salario mensual: ";
    cin >> empleado1.salario;

    if (empleado1.salario < 0)
        empleado1.salario = 0.0;


    cout << "\nIngresa el nombre del empleado 2: ";
    cin >> empleado2.nombre;

    cout << "Ingresa el apellido paterno: ";
    cin >> empleado2.apellido;

    cout << "Ingresa el salario mensual: ";
    cin >> empleado2.salario;

    if (empleado2.salario < 0)
        empleado2.salario = 0.0;


    cout << "\nSalario anual de "
         << empleado1.nombre << ": "
         << empleado1.obtenerSalarioAnual() << endl;

    cout << "Salario anual de "
         << empleado2.nombre << ": "
         << empleado2.obtenerSalarioAnual() << endl;


    empleado1.aumentarSalario();
    empleado2.aumentarSalario();


    cout << "\nDespues del aumento del 10%:" << endl;

    cout << "Salario anual de "
         << empleado1.nombre << ": "
         << empleado1.obtenerSalarioAnual() << endl;

    cout << "Salario anual de "
         << empleado2.nombre << ": "
         << empleado2.obtenerSalarioAnual() << endl;

    return 0;
}